smoketests/basic_auth/basicauth_test.go

```diff
@@ -0,0 +1,112 @@
+package mocking
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/suite"
+	"gopkg.in/yaml.v3"
+	corev1 "k8s.io/api/core/v1"
+
+	// v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Remove if not needed

Reply:
I forgot to turn on teardown which was using v1. 
Good catch